<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindowClass</class>
 <widget class="QWidget" name="AddWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddWindow</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>52</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tytuł:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_title"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Reżyser:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_director"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Kategoria:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Data Obejrzenia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_dates"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Seria:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_series"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Rok powstania:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ocena:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddWindowClass</receiver>
   <slot>acceptAddition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acceptAddition()</slot>
 </slots>
</ui>
